include/circt/Dialect/FIRRTL/Passes.td

```diff
@@ -458,7 +458,8 @@ def GrandCentral : Pass<"firrtl-grand-central", "CircuitOp"> {
                 "Instantiate companions in the design"),
               clEnumValN(CompanionMode::Drop, "drop",
                 "Remove companions from the design"))
-           }]>
+           }]>,
+    Option<"noViews", "no-views", "bool", "false", "Delete FIRRTL views">,
```

Comment:
Nit:
```suggestion
    Option<"noViews", "no-views", "bool", "false", "Delete FIRRTL view intrinsics">,
```

I don't want anybody to think that "Views" are FIRRTL proper...

Reply:
Thanks! Fixed in 8d25d7da90d2f06d83b1587ede40c6934a1c6d74 .